docs/getting_started.md

```diff
@@ -13,34 +13,34 @@ For now Onsho only supports QEMU/KVM. Which makes Linux a requirement. But you s
 
 ## Building Onsho
 
-```
-git clone https://github.com/giantswarm/onsho.git
-cd onsho
-make && sudo make install
+```nohighlight
+$ git clone https://github.com/giantswarm/onsho.git
```

Comment:
afaik we shouldn't add the `$` signs. This makes it impossible to copy paste the snippets.